### PR TITLE
Add RPC `bootstrap`

### PR DESCRIPTION
**Pull request by mhchia**
### What was wrong?
As described in https://github.com/ethresearch/sharding-p2p-poc/issues/108

### How was it fixed?
bootstrap start
```bash
# start bootstrapping with the bootnodes specified in the string 
$ sharding-p2p-poc -client bootstrap start bootnodeStr1,bootnodeStr2,...
# example
$ sharding-p2p-poc -rpcport=13002 -client bootstrap start `/ip4/127.0.0.1/tcp/10000/ipfs/QmS5QmciTXXnCUCyxud5eWFenUMAmvAWSDa1c7dvdXRMZ7,/ip4/127.0.0.1/tcp/10001/ipfs/QmexAnfpHrhMmAC5UNQVS8iBuUUgDrMbMY17Cck2gKrqeX`
```

bootstrap stop
```bash
# start bootstrapping with the bootnodes specified in the string 
$ sharding-p2p-poc -client bootstrap stop
```

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by mhchia** [comment]
@araskachoi does this CLI fits your needs?

**Comment by araskachoi** [comment]
> @araskachoi does this CLI fit your needs?

Yes! this is what was needed for bootstrapping. It was important to have bootstrapping be available even after the initial startup of the application. Thanks alot for this! I will test this out on my environment and see if anything weird comes up!

**Comment by mhchia** [comment]
@araskachoi Thank you a lot for the confirmation:) Will get it ready ASAP

File: rpcserver.go [hunk 1]
```diff
@@ -423,6 +428,53 @@ func (s *server) RemovePeer(
 	return &pbrpc.RPCPlainResponse{}, nil
 }
 
+func (s *server) Bootstrap(
+	ctx context.Context,
+	req *pbrpc.RPCBootstrapRequest) (*pbrpc.RPCPlainResponse, error) {
+	// Add span for RemovePeer
```
Comment (NIC619) [inline rpcserver.go]: nitpick: `Bootstrap`

File: cli.go [hunk 2]
```diff
@@ -121,6 +121,26 @@ func doRemovePeer(rpcArgs []string, rpcAddr string) {
...
+	flagStr := rpcArgs[0]
+	var flag bool
+	var bootnodesStr string
+	if flagStr == "start" {
+		flag = true
+		if len(rpcArgs) == 2 {
+			bootnodesStr = rpcArgs[1]
+		}
+	} else if flagStr == "stop" {
+		flag = false
```
[the remark this answers is not in the document]
Comment (mhchia) [reply cli.go]: I prefer to ignoring them

File: rpcserver.go (hunk 1 above)
Comment (mhchia) [reply rpcserver.go]: Thank you for pointint it out!

File: cli.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (mhchia) [reply cli.go]: Personally I prefer to ignore them

**Pull request merged**